docs/static-pvc.md

```diff
@@ -0,0 +1,240 @@
+  volumeMode: Filesystem
+  # volumeName is static PV name
+  volumeName: fs-static-pv
+```
+
+```bash
+[$] kubectl create -f fs-static-pvc.yaml
+persistentvolumeclaim/fs-static-pvc created
+```
+
+now you can bind this PVC to any app pod which uses filesystem PVC
+
+### Create app pod to use the Filesystem PVC
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: csi-rbd-demo-pod
+spec:
+  containers:
+    - name: web-server
+      image: nginx
+      volumeMounts:
+        - name: mypvc
+          mountPath: /var/lib/www/html
+  volumes:
+    - name: mypvc
+      persistentVolumeClaim:
+        claimName: fs-static-pvc
+```
+
+```bash
+[$] kubectl create -f pod.yaml
+pod/csi-rbd-demo-pod created
+```
+
+wait for app pod to go to running state, now you can consume static PVC for
+storing data
+
+### Delete Resources created for static Filesystem PV
+
+```bash
+[$] kubectl delete pod/csi-rbd-demo-pod
+[$] kubectl delete pvc/fs-static-pvc
+[$] kubectl delete pc/fs-static-pv
+```
+
+*Note* deleting PV and PVC doesnot deleted the backend rbd image, user need to
+manually delete the rbd image if required
+
+### Create static RBD Block PV
+
+to create the rbd PV you need to know the `rbd image name`,`clusterID` and
+`pool` name in which the rbd image is created
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: block-static-pv
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 4Gi
+  csi:
+    driver: rbd.csi.ceph.com
+    fsType: ext4
+    nodeStageSecretRef:
+      # node stage secret name
+      name: rook-csi-rbd-node
+      # node stage secret namespace where above secret is created
+      namespace: rook-ceph
+    volumeAttributes:
+      # cluster id specified in configmap during csi deployment
+      clusterID: abcd
+      imageFeatures: layering
+      imageFormat: "2"
+      pool: replicapool
+      staticVolume: "true"
+    # name of the rbd image created
+    volumeHandle: rbd-test
+  persistentVolumeReclaimPolicy: Retain
+  volumeMode: Block
+```
+
+```bash
+[$] kubectl create -f block-static-pv.yaml
+persistentvolume/block-static-pv created
+```
+
+### Create static RBD Block PVC
+
+to create the rbd PVC you need to know the PV name which is created above
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: block-static-pvc
+  namespace: default
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 4Gi
+  volumeMode: Block
+  # volumeName is static PV name
+  volumeName: block-static-pv
+```
+
+```bash
+[$] kubectl create -f block-static-pvc.yaml
+persistentvolumeclaim/block-static-pvc created
+```
+
+now you can bind this PVC to any app pod which uses block PVC
+
+### Create app pod to use the Block PVC
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-with-raw-block-volume
+spec:
+  containers:
+    - name: fc-container
+      image: fedora:26
+      command: ["/bin/sh", "-c"]
+      args: ["tail -f /dev/null"]
```

Comment:
What does this do? Can you not just use `sleep infinity` instead?

Reply:
This example is copied from https://github.com/ceph/ceph-csi/blob/master/examples/rbd/raw-block-pod.yaml

Reply:
We should improve it there too.

And not use a **very** old fedora image in the references either.

Reply:
See #805 for a better solution :wink: